src/aws/Dockerfile

```diff
@@ -28,6 +31,71 @@ RUN npm install -g yarn
 # configure volume for lambda mediapackge
 VOLUME /mnt/transcoded_video
 
+WORKDIR /root
+
+## Install packages needed to compile poppler
+RUN yum install -y \
```

Comment:
Why did you removed the `yum update` ?

Reply:
Because I added it on a similar purpose than `apt update`, but it's not needed with yum.

Reply:
Yes in the same time I read the yum documentation and it's for updating all packages, not the repo cache.